packages/block-library/src/media-text/constants.js

```diff
@@ -1 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Platform } from '@wordpress/element';
+const isWeb = Platform.OS === 'web';
+
 export const DEFAULT_MEDIA_SIZE_SLUG = 'full';
+export const CSS_UNITS = [
```

Comment:
There has been a [shift away from hard-coding CSS units](https://github.com/WordPress/gutenberg/pull/32482). You can instead use the helpful [`useCustomUnits` hook](https://github.com/WordPress/gutenberg/blob/e9e62d3a10d57b581ca88ca89ee46a88d5d8507d/packages/components/src/unit-control/utils.js#L267) to handle this.